src/main/java/core/basesyntax/service/FruitShopController.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service;
+
+import core.basesyntax.util.Injector;
+
+public class FruitShopController {
+    private static final String INPUT_FILE_PATH = "src/main/resources/reportToRead.csv";
+    private static final String OUTPUT_FILE_PATH = "src/main/resources/finalReport.csv";
+
+    private final FruitShopService fruitShopService;
+
+    public FruitShopController() {
+        this.fruitShopService = Injector.getFruitShopService();
+    }
+
+    public void run() {
+        fruitShopService.processTransactions(INPUT_FILE_PATH, OUTPUT_FILE_PATH);
```

Comment:
Make sure to make all services independent and call them in the `main()` method. The `run` method here seems to be tightly coupling the `FruitShopService` with file paths. Consider refactoring this to follow the SOLID principles more closely.